client/my-sites/domains/domain-management/list/transfer-lock-opt-out.jsx

```diff
@@ -1,41 +0,0 @@
-import { localizeUrl } from '@automattic/i18n-utils';
```

Comment:
Is this case treated in the new domains table? Could we add this test there?

Reply:
I don't quite know what you mean. This component is no longer referenced anywhere, which is why I've removed it.

We do show a transfer lock opt out as part of the bulk contact update form. And it's also shown as part of the single-domain update form. It's just this component that's not used.

Reply:
>We do show a transfer lock opt out as part of the bulk contact update form. And it's also shown as part of the single-domain update form.

What I meant was a way to test the transfer lock opt out in the bulk contact update form.

> This component is no longer referenced anywhere, which is why I've removed it.

Oh, right. I thought it was part of the domains table.